### ISSUE TITLE
Allow disabling of code regions

### ISSUE THREAD

**Issue by Stateford** [PyCQA/pylint#2920]
### Is your feature request related to a problem? Please describe
Disabling pylint for a specific set of code

### Describe the solution you'd like
Something like this might work
```py
# this block does not raise a pylint error

# region 
# pylint: disable=invalid-name
apdex_dir = None
output_files_requested = None
# endregion

# this will raise a pylint error in the same scope
output_files_distribution = None
```
### Additional context
I think this could be useful to allow users to disable certain regions instead of just disabling based on scope. This is useful if the offending region is in the global scope (yes, I know it's bad to have globals)

**Comment by PCManticore** [comment]
@Stateford you can already achieve this using:
```
# pylint: disable=...
...
...
# pylint: enable=...
```
